core/domain/story_services.py

```diff
@@ -201,30 +201,56 @@ def _save_story(committer_id, story, commit_message, change_list):
     story.validate()
     # Validate that all explorations referenced by the story exist.
     exp_ids = []
+
+    # Any exp ID in the story to compare categories later on.
```

Comment:
This seems like a sentence fragment; I don't understand it. Could you please formulate this as a full sentence?

Reply:
Ah -- looks like you want to get some non-empty exp ID. Don't do it like this. Just take exp_ids[0] after you've calculated everything.

You can probably simplify a lot of this to:
```
exp_ids = [node.exploration_id for node in story.story_contents.nodes if node.exploration_id is not None]
sample_exp_id = exp_ids[0] if exp_ids else None
```

Reply:
Done